app/models/jws.py

```diff
@@ -26,8 +26,20 @@ def check_at_least_one_field_is_populated(cls, values):
         did, verification_method = values.get("did"), values.get("verification_method")
         if not did and not verification_method:
             raise CloudApiValueError(
-                "At least one of `did` or `verification_method` must be populated."
+                "One of `did` or `verification_method` must be populated."
             )
+        if did and verification_method:
+            raise CloudApiValueError(
+                "Only one of `did` or `verification_method` can be populated."
+            )
```

Comment:
I'm taking your word for it that this is true :-) I'm not familiar with how it's implemented, and I thought there was test coverage where both are provided, but perhaps not